### PR TITLE
Add new SE Embark flows

### PR DESCRIPTION
**Pull request by gustaveen**
## What?

- Add new SE Embark flows
- Bump brand and embark package
- Add naive implementation of `VariantedOfferRedirect`. This makes it possible to go through the embark flow and get an offer

## Why 
Being able to test the flows in staging

### Review app:
SE-SV Needer https://web-onboardi-feature-ad-y5grim.herokuapp.com/se/new-member/home-accident-needer
SE-EN Needer https://web-onboardi-feature-ad-y5grim.herokuapp.com/se-en/new-member/home-accident-needer

### CODE REVIEW

File: src/routes.tsx [hunk 1]
```diff
@@ -253,6 +256,16 @@ export const reactPageRoutes: ReactPageRoute[] = [
...
+              case 'home-accident-needer':
+                return {
+                  baseUrl: `/${locale}/new-member/home-accident-needer`,
+                  name: EmbarkStory.SwedenNeeder,
+                }
+              case 'home-accident-switcher':
+                return {
+                  baseUrl: `/${locale}/new-member/home-accident-switcher`,
+                  name: EmbarkStory.SwedenSwitcher,
+                }
```
Comment (mehdielket) [inline src/routes.tsx:268]: Is the plan to replace to other Swedish flows with the new one? If so, when would we be able to do that?

Comment (gustaveen) [reply src/routes.tsx:268]: Correct! Probably the last thing we do before going live :)

**Review by robinandeer — DISMISSED** [review]
Looks good to me!

**Review by robinandeer — APPROVED** [review]
Looks good, great job!

**Pull request merged**